packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -184,6 +192,13 @@ export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> imp
         return this.container.connected;
     }
 
+    /**
+     * {@inheritDoc IFluidContainer.connectionState}
+     */
+     public get connectionState(): ConnectionState {
```

Comment:
What's our strategy RE renames within ConnectionState? It's probably fine to do all changes after the fact...

Reply:
The rename for `ConnectionState` is a breaking change, so it will need to be in the next branch. Since `connectionState` is the intended replacement for `connected` we would have to wait to deprecate it until the rename is complete. Do you think this would be preferred?

Reply:
I think the `ConnectionState` rename may not be a breaking change, looking at how `ConnectionState` is defined (both types in container-definitions and enum container-loader packages).

Because in fact there are no names at runtime (just 0, 1, 2), and in both places you can have multiple names for the same numerical value. And since we are not planning to reuse/repurpose existing names, we can just add the new ones and leave the old ones.

Still good to do it in a later PR.  If we get agreement on https://github.com/markfields/FluidFramework/pull/2/files then it's just changing `Connecting` to `Pending`.